diff --git a/core/themes/engines/twig/twig.engine b/core/themes/engines/twig/twig.engine
index a3d2a5e26d..342e2343e9 100644
--- a/core/themes/engines/twig/twig.engine
+++ b/core/themes/engines/twig/twig.engine
@@ -14,7 +14,8 @@ use Drupal\Core\Extension\Extension;
  */
 function twig_theme($existing, $type, $theme, $path) {
   $templates = drupal_find_theme_functions($existing, [$theme]);
-  $templates += drupal_find_theme_templates($existing, '.html.twig', $path);
+  module_load_include('module', 'site_module');
+  $templates += site_module_find_theme_templates($existing, '.html.twig', $path);
   return $templates;
 }
 
